lib/findJsModulesFor.js

```diff
@@ -50,6 +50,12 @@ function findJsModulesFromModuleFinder(
         const modules = exports.map((
           { name, path, isDefault, packageName }: Object,
         ): ?JsModule => {
+          // Filter out modules that are in the `excludes` config.
+          if (config.get('excludes').some((glob: string): boolean =>
```

Comment:
nit: I think this would be more readable if you used a variable here. something like this:

```js
const isExcluded = config.get('excludes')
  .some((glob: string): boolean => minimatch(path, glob));
if (isExcluded) {
  return undefined;
}
```

It looks like you are just moving this code from somewhere else!